pkg/controller/controller_binding_test.go

```diff
@@ -82,8 +82,9 @@ func TestReconcileServiceBindingNonExistingServiceInstance(t *testing.T) {
 	events := getRecordedEvents(testController)
 	assertNumEvents(t, events, 1)
 
-	expectedEvent := corev1.EventTypeWarning + " " + errorNonexistentServiceInstanceReason + " " + "References a non-existent ServiceInstance \"/nothere\""
-	if err := checkEvents(events, []string{expectedEvent}); err != nil {
+	expectedEvent := new(MessageBuilder).warning().reason(errorNonexistentServiceInstanceReason)
+	expectedEvent.msgf("References a non-existent ServiceInstance %q", "/"+testNonExistentClusterServiceClassName)
+	if err := checkEvents(events, expectedEvent.stringArr()); err != nil {
```

Comment:
why `stringArr` instead of just `String()`?

Reply:
it needs to be a []string